emission/net/api/cfc_webapp.py

```diff
@@ -379,6 +382,41 @@ def getCarbonCompare():
     logging.debug("No overriding client result for user %s, returning choice" % user_uuid)
   return choice.getResult(user_uuid)
 
+
+#Pulling public data from the server  
+@get('/getData')
```

Comment:
this is the raw, unprocessed data.
I store the results of the processed data in a separate timeseries collection so we could have `eval/publicData/timeseries` and `eval/publicData/analysisTimeSeries` or something.